src/csharp/Microsoft.Spark/ML/Feature/StopWordsRemover.cs

```diff
@@ -0,0 +1,173 @@
+        /// <see cref="StopWordsRemover"/> a unique ID
+        /// </summary>
+        /// <param name="uid">An immutable unique ID for the object and its derivatives.</param>
+        public StopWordsRemover(string uid) : base(s_stopWordsRemoverClassName, uid)
+        {
+        }
+
+        internal StopWordsRemover(JvmObjectReference jvmObject) : base(jvmObject)
+        {
+        }
+
+        JvmObjectReference IJvmObjectReferenceProvider.Reference => _jvmObject;
+
+        /// <summary>
+        /// Sets the column that the <see cref="StopWordsRemover"/> should read from.
+        /// </summary>
+        /// <param name="value">The name of the column to as the source</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetInputCol(string value) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setInputCol", value));
+
+        /// <summary>
+        /// The <see cref="StopWordsRemover"/> will create a new column in the DataFrame, this is the
+        /// name of the new column.
+        /// </summary>
+        /// <param name="value">The name of the column to as the target</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetOutputCol(string value) =>
+            WrapAsStopWordsRemover(_jvmObject.Invoke("setOutputCol", value));
+
+        /// <summary>
+        /// Executes the <see cref="StopWordsRemover"/> and transforms the DataFrame to include the new
+        /// column.
+        /// </summary>
+        /// <param name="source">The DataFrame to transform</param>
+        /// <returns>
+        /// New <see cref="DataFrame"/> object with the source <see cref="DataFrame"/> transformed
+        /// </returns>
+        public DataFrame Transform(DataFrame source) =>
+            new DataFrame((JvmObjectReference)_jvmObject.Invoke("transform", source));
+
+        /// <summary>
+        /// Gets the column that the <see cref="StopWordsRemover"/> should read from.
+        /// </summary>
+        /// <returns>Input column name</returns>
+        public string GetInputCol() => (string)_jvmObject.Invoke("getInputCol");
+
+        /// <summary>
+        /// The <see cref="StopWordsRemover"/> will create a new column in the DataFrame, this is the
+        /// name of the new column.
+        /// </summary>
+        /// <returns>The output column name</returns>
+        public string GetOutputCol() => (string)_jvmObject.Invoke("getOutputCol");
+
+        /// <summary>
+        /// Sets locale for <see cref="StopWordsRemover"/> transform.
+        /// Refer java.util.locale.getavailablelocales() for all available locales.
+        /// </summary>
+        /// <param name="value">Locale to be used for transform</param>
+        /// <returns>New <see cref="StopWordsRemover"/> object</returns>
+        public StopWordsRemover SetLocale(string value) =>
```

Comment:
SetLocale and GetLocale are only available since 2.4.0, the tests have the Skip attribute but we also need to add the `[Since(Versions.V2_4_0)]` attribute to the methods.